counterparty-core/counterpartycore/test/gas_test.py

```diff
@@ -0,0 +1,97 @@
+import tempfile
```

Comment:
let's also add a test that makes sure the values are right (prints out the fee at lots of different tx levels so we can plot it easily to verify the behavior)

Reply:
done